benchmark/citation/datasets.py

```diff
@@ -1,11 +1,11 @@
-import os.path as osp
+from pathlib import Path
 
 import torch_geometric.transforms as T
 from torch_geometric.datasets import Planetoid
 
 
 def get_planetoid_dataset(name, normalize_features=False, transform=None):
-    path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', name)
+    path = Path.joinpath(Path(__file__).resolve().parent.parent, 'data', name)
```

Comment:
I think this would be a little bit cleaner.
```suggestion
    path = Path(__file__).resolve().parent.parent.joinpath('data', name)
```

Reply:
In the previous PR a reviewer(cthoyt) suggested me to use second parent

Reply:
Yes, we need two parents here (one for getting the `dirname` and one for going one step back). Just wanted to point out that I think the beauty of `Path` is that we can combine paths via regular slashes. In the end, `Path.joinpath()` syntax is not really needed since we can directly do this, e.g., via `root_path / '..' / 'data'`.